go/teams/audit.go

```diff
@@ -146,6 +146,12 @@ func (a *Auditor) AuditTeam(m libkb.MetaContext, id keybase1.TeamID, isPublic bo
 			Ctime:     keybase1.ToTime(start),
 		})
 	}()
+	defer func() {
```

Comment:
@maxtaco do you want me to change this one too? The defer above this one can also be simplified, so not sure what the best thing to do is....

Reply:
urg, some of the ugliness is pre-existing here. In general, I'd say two use a `defer` if you'd otherwise need to insert that code at `>1` callsites. Which as you point out isn't the case here.